### ISSUE TITLE
[UoM] Main UI Color Temperature Slider displays Celsius rather than Kelvin

### ISSUE THREAD

**Issue by andrewfg** [openhab/openhab-webui#1913]
See https://github.com/openhab/openhab-core/issues/3643 where @J-N-K says it is not an OH Core problem.

On OH 4.0 snapshot, for things with channels resp. items of type system.color-temperature-abs the Main UI default color temperature slider for those items displays the values using the system default temperature measurement Celsius (as shown below). Expected would be that Main UI would display color temperature in Kelvin (even in the USA)..

(It also sends a QuantityType(Celsius) to the binding when the slider is moved. Which is not a problem since the binding can convert the UoM to Kelvin).

I suppose that this is related to recent UoM changes in OH core??

![image](https://github.com/openhab/openhab-webui/assets/893994/551a8b78-5c39-4e61-89c8-9854f20e8056)

Note: I did not check on a US system but I suppose the default color temperature slider would display in Fahrenheit..

**Comment by florian-h05** [comment]
Can you please share the REST response for that Item with me, or if you don‘t find it, your browsers network traffic?

**Comment by J-N-K** [comment]
Since the correct value is reported in the item view (lower part), it seems that the UI uses the raw state for rendering the slider.

**Comment by florian-h05** [comment]
Yes, took me a second to notice that. I will try to fix that soon, I guess oh-stepper and oh-knob also need to be adjusted.
